build/archives/BUILD.gn

```diff
@@ -20,6 +20,7 @@ if (build_engine_artifacts) {
       prefix = "$full_platform_name-$flutter_runtime_mode/"
     }
     output = "$prefix/artifacts.zip"
+    encode_entitlements = false
```

Comment:
Do we need this? or can we rely on the entitlements and without_entitlements list?